packages/browser-agent-core/common/wrap/wrap-xhr.js

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// wrap-events patches XMLHttpRequest.prototype.addEventListener for us.
+import './wrap-events'
```

Comment:
I don't think this will guarantee that events are wrapped. Other features call wrap-events, so for Lite, Pro, SPA this should work. But the Ajax feature on its own assumes that `import './wrap-events'` does the same thing as the previous `require('./wrap-events')` and auto-wraps.

To be safe we may want to call wrapEvents() around line 22.

This is a dependency that maybe we want to represent somewhere more unified as well.

Reply:
good catch